### PR TITLE
onstop support

### PR DESCRIPTION
**Pull request by zapion**
Support for graceful shutdown

### CODE REVIEW

**Comment by zapion** [comment]
@ShakoHo @askeing r?

File: moz_minions/minions.py [hunk 1]
```diff
@@ -88,6 +91,13 @@ def _work(self, **kwargs):
             "%s's worker function is not yet implemented." % (self.__class__)
             )
 
+    def on_stop(self):
```
Comment (ShakoHo) [inline moz_minions/minions.py]: typo?  boss.py:133  .onstop()   vs  on_stop() here

Comment (zapion) [inline moz_minions/minions.py]: ya...need test to cover

**Comment by ShakoHo** [comment]
lgtm r+

**Pull request merged**